.github/workflows/deploy.yml

```
@@ -0,0 +1,11 @@
+name: Deploy
+
+on:
+  branches:
+    - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)